index.js

```diff
@@ -26,10 +26,16 @@ module.exports = function(dockerFile){
 		arrayFieldMap[f] = true
 	})
 
+        dockerFile = dockerFile.replace(/\\\n/g, '')
```

Comment:
Small thing I'm confused about - why are we removing all newlines here?
Does this not make splitting on newlines further down invalid?